test/integration/integration_test.go

```diff
@@ -0,0 +1,192 @@
+// +build integration
```

Comment:
A style question: in other places in boulder we've succumbed to having giant files with tons of stuff packed into them which can make finding a thing you are looking for a huge pain. How would you feel about splitting some of this stuff out into separate files? i.e. having a `helpers.go` and then _perhaps_ one file per integration test (since some tests can get rather complicated and long)? I think the main trade off here is that we'd end up with lots of little files vs. one gigantic one, but I think that might end up preferable.

Reply:
I'm definitely fine with that. Will rename this to ocsp_test.go in anticipation of sharing future space with other OCSP tests.